provision.yml

```diff
@@ -67,11 +66,12 @@
 
 - hosts: proxy
   roles:
-    - { role: satosa,      tags: ['proxy','satosa'    ] }
-    - { role: pyff,        tags: ['proxy','pyff'      ] }
-    - { role: cmservice,   tags: ['proxy','cmservice' ] }
-    - { role: mysql,       tags: ['proxy','mysql'     ] }
-    - { role: proxy_mysql, tags: ['proxy','mysql'     ] }
+    - { role: syslog-client, tags: ['proxy','syslog'    ] }
+    - { role: satosa,        tags: ['proxy','satosa'    ] }
+    - { role: pyff,          tags: ['proxy','pyff'      ] }
+    - { role: cmservice,     tags: ['proxy','cmservice' ] }
+    - { role: mysql,         tags: ['proxy','mysql'     ] }
+    - { role: proxy_mysql,   tags: ['proxy','mysql'     ] }
 
 - hosts: comanage
```

Comment:
Could you also add a custom logreader for comanage (for `/var/www/comanage-registry-3.0.0/local/tmp/logs/*`)?